www/js/app.js

```diff
@@ -995,9 +995,9 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
     
     // Compile some regular expressions needed to modify links
     // Pattern to find the path in a url
-    var regexpPath = /^(.*\/)[^\/]+$/;
+    // var regexpPath = /^(.*\/)[^/]+$/; // Kept for reference
```

Comment:
Why do you keep this commented line? (and not for the other similar modifications?)

Reply:
Codefactor identified this line as redundant: we never use `regexPath` any more in our code. I thought, however, that it might be useful to have the regex in the code (but commented out) for reference, in case we ever need it. However, I can delete. It's not exactly a hard regex to write if we need it again.

Reply:
You can remove it, as it's unused.
At worst, we still have it in git history.